lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -326,4 +333,85 @@ class LineChartPainter extends AxisChartPainter {
 
   @override
   bool shouldRepaint(CustomPainter oldDelegate) => false;
+
+  void drawAnnotation(Canvas canvas, Size viewSize) {
+    if (data.extraLinesData != null && data.extraLinesData.show) {
+      if (data.extraLinesData.horizontalLines != null) {
+        for (HorizontalLine line in data.extraLinesData.horizontalLines) {
+          _drawHorizontalLineLine(canvas, viewSize, line);
+        };
+      }
+      if (data.extraLinesData.verticalLines != null) {
+        for (VerticalLine line in data.extraLinesData.verticalLines) {
+          _drawVerticalLineLine(canvas, viewSize, line);
+        }
+      }
+      return;
+    }
+  }
+
+  void _drawHorizontalLineLine(Canvas canvas, Size viewSize, HorizontalLine line) {
+    viewSize = getChartUsableDrawSize(viewSize);
+    extraLinesPaint.color = line.color;
+    extraLinesPaint.strokeWidth = line.strokeWidth;
+    final double x = line.endX == null ? viewSize.width : getPixelX(line.endX, viewSize);
```

Comment:
here we can equal x with the viewSize.width every time.
because this is not for draw below lines

Reply:
Removed